modules/overview/pages/tokens.adoc

```diff
@@ -0,0 +1,154 @@
+include::{mod-loc}attributes.adoc[]
+
+:context: skupper-tokens
+//Category: skupper-tokens
+// Type: assembly
+[id='using-skupper-tokens-{context}']
+
+= Using Skupper tokens
+
+[role="system:abstract"]
+Skupper tokens allow you create links between sites.
+You create a token on one site and use that token from the other site to create a link between the two sites.
+
+NOTE: Although the linking process is directional, a Skupper link allows communication in both directions.
+
+If both sites are equally accessible, for example two public clouds, then it not important where the token is created.
+However, when using a token, the site you are linking to must be accessible from the site you are linking from.
+For example, if you are creating a {service-network} using both a public and private cluster, you must create the token on the public cluster and use the token from the private cluster.
+
+There are two types of Skupper token:
+
+Claim token (default):: 
++
+--
```

Comment:
Maybe a note here to clarify that a claim token will be automatically exchanged for a certificate by the linking site, preventing the certificate from needing to be exposed at all.

Reply:
added  text